torch/_inductor/codegen/cpp.py

```diff
@@ -208,6 +208,10 @@ def cos(x):
     def exp(x):
         return f"{x}.exp()"
 
+    @staticmethod
+    def erf(x):
```

Comment:
I see https://github.com/pytorch/pytorch/pull/89388 didn't add `erf` here. This raises a general question: people changing the cpp backend may easily miss here, or not sure about if vectorization applies for the new op. Can we have a better way to avoid this kind of problem in future (I think Jason asked about using inheritance in one of the initial vectorization PRs)?

Reply:
That's why @EikanWang added a UT to make sure the op list is the same for explicit vec and omp vec here:
https://github.com/pytorch/pytorch/pull/89837/files#diff-809c39aeafb3acc92289f42a63e670a8719d4ce5627d5f88820142d80edf8d2aR4950